assets/schema/ensemble_schema.json

```diff
@@ -42,6 +42,10 @@
                   "$ref": "#/$defs/typeColors",
                   "description": "By default the background color uses the theme's 'primary' color. You can override the header's background color here."
                 },
+                "onScrollBackgroundColor": {
```

Comment:
The onScroll* name doesn't make sense. Not that surfaceTintColor makes sense either but at least it's the same name Flutter uses.